challenge_2/c/almyk/src/main.c

```diff
@@ -0,0 +1,66 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+#define MAX_SIZE 128
```

Comment:
You are right, I intended to use it but changed my approach and missed to remove this